xgi/generators/classic.py

```diff
@@ -98,3 +99,40 @@ def empty_simplicial_complex(create_using=None, default=None):
     if default is None:
         default = xgi.SimplicialComplex
     return _empty_network(create_using, default)
+
+
+def trivial_hypergraph(create_using=None, default=None):
+    """Returns a hypergraph with one node and zero edges.
```

Comment:
Could you make this a more flexible function so that it creates a hypergraph with n nodes and zero edges, where 1 node is the default or something like that?